sdk/javascript/src/BaseValue.js

```diff
@@ -39,6 +39,14 @@ class BaseValue {
 		this.isSigned = isSigned;
 		this.value = check(size, value, isSigned);
 	}
+
+	/**
+	 * Converts base value to string.
+	 * @returns {string} String representation.
+	 */
+	toString() {
+		return this.value.toString();
```

Comment:
we should probably make this fixed width and in hex...

Reply:
probably